src/SkynetServer/Network/Packets/P1CDirectChannelCustomization.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SkynetServer.Model;
+using SkynetServer.Network.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SkynetServer.Network.Packets
+{
+    [Packet(0x1C, PacketPolicies.Duplex)]
+    [MessageFlags(MessageFlags.Loopback)]
+    internal sealed class P1CDirectChannelCustomization : ChannelMessage
```

Comment:
Do we need empty packet classes for packets the server can't handle due to encryption?

Reply:
Yes, we need these empty classes in order to enforce `MessageFlags`.